lib/utils.js

```diff
@@ -118,6 +118,20 @@ const utils = {
   },
 
   /**
+   * Grab the most recent PR
+   * @param {Config} config - the config object
+   * @param {Github} vcs - VCS interface
+   * @returns {Promise} a promise resolved with the most recent PR
+   */
+  getLastPr (config, vcs) {
+    return exec(`git log -50 --oneline | grep -m 1 "Merge pull request" | awk '{print $5;}'`).then((stdout) => {
```

Comment:
How difficult do you think it would be to do some of this in JavaScript or someone split it into multiple lines for commenting what the three piped commands do?